### PR TITLE
Add description to empty feed

### PR DESCRIPTION
**Pull request by asf-algoo**
## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [ ] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [ ] Manual, quality tests have been done

### CODE REVIEW

File: frontend/src/container/Publications.jsx [hunk 1]
```diff
@@ -505,6 +506,10 @@ export class Publications extends React.Component {
           itemList={props.publicationList}
           shouldScrollToBottom={currentPublicationId === 0}
         >
+          {isPublicationListEmpty
+            ? <div className='publications__empty'>{props.t('This space does not have any publications yet, create the first publication using the area at the bottom of the page.')}</div>
+            : null}
```
Comment (gbonaspetti) [inline frontend/src/container/Publications.jsx:511]:
```suggestion
          {isPublicationListEmpty && (
            <div className='publications__empty'>
              {props.t('This space does not have any publications yet, create the first publication using the area at the bottom of the page.')}
            </div>
          )}
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend/i18next.scanner/fr/translation.json [hunk 2]
```diff
@@ -442,5 +442,6 @@
   "copied": "copié",
   "You have reached your storage limit, you cannot add new files": "Vous avez atteint votre limite de stockage, vous ne pouvez pas ajouter de nouveaux fichiers",
   "Download {{filename}}": "Télécharger {{filename}}",
-  "Show in publications": "Afficher dans les publications"
+  "Show in publications": "Afficher dans les publications",
+  "This space does not have any publications yet, create the first publication using the area at the bottom of the page.": "Cet espace n'a pas encore de publications, créez la première publication en utilisant la zone en bas de la page."
```
Comment (raphj) [inline frontend/i18next.scanner/fr/translation.json:446]:
```suggestion
  "This space does not have any publications yet, create the first publication using the area at the bottom of the page.": "Cet espace n'a pas encore de publication, créez la première publication en utilisant la zone en bas de la page."
```

**Review by raphj — APPROVED** [review]

File: frontend/src/container/Publications.jsx (hunk 1 above)
Comment (MariePrandstetter) [reply frontend/src/container/Publications.jsx]: I added the suggestion

File: frontend/i18next.scanner/fr/translation.json (hunk 2 above)
Comment (MariePrandstetter) [reply frontend/i18next.scanner/fr/translation.json]: I added the suggestion

Comment (asf-algoo) [reply frontend/i18next.scanner/fr/translation.json]: fixed

**Pull request merged**